scripts/sbom_scraper.sh

```diff
@@ -133,19 +159,179 @@ SECRET=$(cat "${CLIENTSECRET_FILE}")
 # ----------------------------------------------------------------------------
 if [ "${SBOM}" = "false" ]
 then
-    log "Scrape ${FORMAT} SBOM from ${DOCKER_IMAGE} to ${OUTFILE}..."
+    log "Scrape ${FORMAT} SBOM from ${DOCKER_IMAGE} to ${OUTFILE} ..."
     OUTPUT="${TEMPDIR}/${OUTFILE}"
     ${SYFT} -q packages -o "${FORMAT}" "${DOCKER_IMAGE}"> "${OUTPUT}"
 else
     OUTPUT="${DOCKER_IMAGE}"
 fi
 
+# ----------------------------------------------------------------------------
+# Update SBOM including NTIA minimum elments
+# ----------------------------------------------------------------------------
+ORIG_COMPONENT_NAME=$(xq -r .bom.metadata.component.name "$OUTPUT")
```

Comment:
whwre does xq come from? its not checked at top of file?

Reply:
added check, thanks